firmware/version_data.py

```diff
@@ -0,0 +1,117 @@
+import  subprocess
```

Comment:
Please use Python 3, not Python 2.

Please make the script executable and have a correct [hash bang](https://en.wikipedia.org/wiki/Shebang_(Unix)) at the top.

Reply:
This file still isn't Python 3.